src/Providers/CloudWatchServiceProvider.php

```diff
@@ -114,11 +114,17 @@ protected function getCredentials()
             throw new IncompleteCloudWatchConfig('Missing region key-value');
         }
 
-        return $awsCredentials = [
+        $awsCredentials = [
             'region' => $cloudWatchConfigs['region'],
             'version' => $cloudWatchConfigs['version'],
-            'credentials' => $cloudWatchConfigs['credentials'],
         ];
+        
+        // If your server has access to cloudwatch through the EC2 role, leave key empty
```

Comment:
you can leave this comment. I think the code is self explanatory. This can be a great addition to readme though.

Reply:
Updated readme